### PR TITLE
Added Colin Charles as a Team Member

### PR DESCRIPTION
**Pull request by abhinavk96**
Added him as a team member along with linkedin and twitter links.

### CODE REVIEW

**Comment by jigyasa-grover** [comment]
@CosmicCoder96 Could you squash your commit into one and rename it as "Fixes #7 : < commit message >" ?

**Comment by abhinavk96** [comment]
@jig08  I have squashed it into a single commit.

File: index.html [hunk 1]
```diff
@@ -915,13 +915,35 @@ <h1 class="text-white">FOSSASIA is the annual OpenTechSummit featuring prominent
...
+
+                </div>
 
             </section>
 
             <a id="sponsors" class="in-page-link"></a>
 
             <section class="sponsors">
-                <div class="container">
+                <div class="" lass="container">
```
Comment (jigyasa-grover) [inline index.html]: @CosmicCoder96 This.

**Comment by jigyasa-grover** [comment]
@CosmicCoder96 It still shows as 2 for me :(

**Comment by abhinavk96** [comment]
@jig08 sorry for the typo , I will fix it. Thank you for the patience. I'm new to contributing.

**Comment by jigyasa-grover** [comment]
@CosmicCoder96 No worries. You can edit in this PR itself .

**Pull request closed without merging**